src/xbot/common/command/BaseRobot.java

```diff
@@ -64,12 +68,14 @@ public void robotInit() {
             DOMConfigurator.configure("log4j.xml");
         } catch (Exception e) {
             // Had a problem loading the config. Robot should continue!
-            System.out.println("Couldn't configure logging - file probably missing or malformed");
+            log.error("Couldn't configure logging - file probably missing or malformed");
```

Comment:
I think using System.out here was intentional since the `log` might not go anywhere in this case. Maybe do both?